GOVERNANCE.md

```diff
@@ -11,21 +11,46 @@ The Fluentd community adheres to the following principles:
 
 ## Voting
 
-The Fluentd project employs "organization voting" to ensure no single organization can dominate the project.
+The Fluentd project employs "Organization Voting" and "Vetoes" to ensure no single organization can dominate the project.
+
+For formal votes, we follow these steps.
+
+1. Have enough period to discuss the topic so that all discussion members agree that there are no more points to be discussed.
+2. Add a specific statement of what is being voted on to the relevant GitHub issue or PR.
+3. Declare the start of voting.
+4. Ask maintainers to indicate their yes/no vote on the issue or PR.
+5. Tally the votes and note the outcome after a suitable period.
```

Comment:
Maybe _outcome_ should be plural: outcome**s**.

Reply:
I will fix it! Thanks!

Reply:
Fixed.